frontend/src/components/CommonSidebar.vue

```diff
@@ -58,7 +65,8 @@
 </template>
 
 <script setup lang="ts">
-import { computed, VNode } from "vue";
+import { ChevronDown, ChevronUp } from "lucide-vue-next";
```

Comment:
Maybe `👇 ChevronDown` and `👉 ChevronRight`?

![CleanShot 2023-10-27 at 10 41 24](https://github.com/bytebase/bytebase/assets/24653555/ef52e015-6170-42bf-a34b-ebaf6b97b717)

Reply:
Just follow the GCP for now